src/Features/Core/Portable/GenerateConstructorFromMembers/ConstructorDelegatingCodeAction.cs

```diff
@@ -71,9 +73,10 @@ private class ConstructorDelegatingCodeAction : CodeAction
                         parameters: _state.Parameters,
                         statements: statements,
                         thisConstructorArguments: thisConstructorArguments),
-                    new CodeGenerationOptions(contextLocation: syntaxTree.GetLocation(_state.TextSpan)),
-                    cancellationToken: cancellationToken)
-                    .ConfigureAwait(false);
+                    new CodeGenerationOptions(
+                        contextLocation: syntaxTree.GetLocation(_state.TextSpan),
+                        afterThisLocation: _state.TextSpan.IsEmpty ? syntaxTree.GetLocation(_state.TextSpan) : null),
```

Comment:
yeah, i should def comment ths :)  

General idea: if you're selecting a bunch of members, and generating a constructor, then we'll auto find the right place for the constructor.

However, if you're just invoking the feature at a particular location, then we'll insert at that location point.